dvc/api.py

```diff
@@ -13,12 +13,15 @@
 from dvc.exceptions import DvcException, NotDvcRepoError
 from dvc.external_repo import external_repo
 
+DEF_SUMMON = "Summon.yaml"
```

Comment:
minor: make it lower case? :) the only example is Makefile? and obviously, not the best example ...

Reply:
Agree. But then we need to consider renaming default repro goal - `Dvcfile`. Or accept two default options `Dvcfile` and `dvcfile`.

Reply:
@dmpetrov it's `Dockerfile`, but `docker-compose.yml`. Basically everything is lowercase if it doesn't resemble Makefiles. Also, why not `dvcsummon.yaml`?

Reply:
I agree with @Suor , plus we have `.dvcignore` already as an example of dvc-specific file :slightly_smiling_face:

Reply:
Let's keep `dvcsummon.yaml`